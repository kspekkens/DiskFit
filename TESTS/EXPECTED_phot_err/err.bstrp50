EXAMPLE/PHOT/phot_err.inp                                                                           
    lpa:  T    leps: T lcentre: T lsystemi F lradial: F lnax:    T luncert: T
#nunc   npars    ntot   nellip  nminr  nmaxr seed
      5      9     28     19      1      8    -50
 
 Bootstrap number:            0
   2.1816616058349609       0.40000000596046448        130.00000000000000        130.00000000000000        1.6580628156661987       0.50000000000000000       0.20000000298023224        2.0000000000000000        10.000000000000000     
   648.506470       634.218445       528.169250       442.703857       365.753845       297.136688       251.127014       200.735275       166.120956       134.525131       111.570068       89.4348831       74.4827576       60.1563950       49.5076180       38.7399483       29.2499657       7.84063816      0.669347703       190.584396       173.984665       198.761490       207.668594       181.336105       247.872604       65.1590118       0.00000000       365.154236    
  0.799456537       9.84473452E-02  0.102096096    

 Bootstrap number:     1/    5  Chisq      1.07941
   2.2693159406316292       0.28717973019723825        129.98798485601614        129.99075675784405        1.7561031784240986       0.59101279429664955       0.19226765034602805        1.6630777152678182        8.2761827292135255     
   649.186768       643.470032       528.828064       435.199219       361.356171       294.220520       251.496292       203.267181       163.181778       133.898636       110.383713       88.3185577       74.3656082       59.9433937       48.2801514       41.5760040       26.4428310       8.93905258       0.00000000       202.256592       156.311935       199.683548       213.084183       186.150696       253.061401       68.6820602       0.00000000       375.203217    
  0.79964077472686768        9.9893346428871155E-002  0.10046585649251938     

 Bootstrap number:     2/    5  Chisq      1.07175
   2.2821713122654610       0.29205879856124434        129.99270686799520        129.98117944404032        1.7538292815196870       0.59105112614708832       0.17228396752140207        1.7143337648322456        8.3884311660156268     
   661.382141       628.557861       523.512695       432.447662       362.403046       291.752655       249.565521       201.128937       166.223557       133.508743       111.248787       87.0586929       78.6461868       57.6456985       52.1854134       38.0180626       29.6426888       8.19199657      0.453900903       221.927795       185.850784       207.065186       215.051895       202.071686       254.651382       77.0176392       0.00000000       349.037292    
  0.79529637098312378       0.10497625917196274        9.9727362394332886E-002

 Bootstrap number:     3/    5  Chisq      1.10209
   2.2602839589334081       0.29504224778140814        129.99044999052322        130.00311332791298        1.7445184157653710       0.58769386159079229       0.19042318606999015        1.7342847369808960        10.650269822739038     
   472.884735       534.944092       466.581940       405.260345       344.516144       280.468781       238.919693       196.365921       162.488586       132.846985       109.777847       87.7666092       73.3945465       62.5621605       48.2600174       37.8064804       29.4526672       8.10936069       1.99429905       188.981369       163.099960       192.182907       201.245636       175.820953       247.524536       56.7215385       0.00000000       329.773499    
  0.75551742315292358        9.5668524503707886E-002  0.14881405234336853     

 Bootstrap number:     4/    5  Chisq      1.08975
   2.2546960732731716       0.29481080366554052        129.99255834234646        129.99919234947177        1.7491216088506216       0.59361433285610743       0.15514559628480282        1.6005747261627983        6.8092386086460053     
   819.309631       724.067932       583.629333       471.236969       385.212250       305.342102       255.920410       201.321350       170.574020       134.154312       113.925331       89.6366730       74.3926086       61.0740700       50.6334648       38.5002899       29.6565018       6.74208307       1.69349349       165.906555       164.381531       176.450439       198.816376       172.075882       247.796066       63.4423218       0.00000000       395.652130    
  0.83229488134384155        9.3749567866325378E-002   7.3955580592155457E-002

 Bootstrap number:     5/    5  Chisq      1.09125
   2.2684700103560691       0.30094542029013571        129.98492330784018        130.00020323476389        1.7451141077366723       0.59030155188285027       0.15038026293569018        1.6363469530570722        6.9407895739829133     
   796.417847       705.441711       570.679016       464.656250       378.629883       307.600800       255.066788       205.618912       169.382782       137.713974       114.358665       91.3167267       72.2761993       62.9001007       52.4445457       38.5825729       29.9188862       9.05879116       1.41080976       181.677429       179.648758       194.921463       207.492538       181.666565       246.979523       66.3048019       4.37268209       384.700348    
  0.82525455951690674        9.8875418305397034E-002   7.5870014727115631E-002

