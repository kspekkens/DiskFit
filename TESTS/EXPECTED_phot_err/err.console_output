 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/PHOT/phot_err.inp                                                                           

Input disk eps: (1-b/a)  0.40
Input disk PA (N->E; deg) =  35.00
Input x-center (data units) =   130.00
Input y-center (data units) =   130.00
Sky, sky sig, gain (ADU) =   800.00   14.14    4.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 Bar profile
 Bar PA
 Bar ellipticity
 Sersic bulge profile
 bulge effective radius
 bulge ellipticity
 bulge Sersic index
 --> Other attributes fixed to input values

 Surface brightnesses extracted at radii:
    5.0   10.0   15.0   20.0   25.0   30.0   35.0   40.0   45.0   50.0
   55.0   60.0   65.0   70.0   75.0   80.0   90.0  100.0  110.0

 Fitting for a bar component
 Axisymmetric model outside    40.0000000    

 Read in photometric image from:
 EXAMPLE/PHOT/phot.fits

 Number of pixels in selected part of the image:       65536
 Number of pixels used in fit:        7495
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.838E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 9.880E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7303  0.306540  129.9455  129.9945   11.2600    0.5840    0.1452    1.9261   10.2436
starting iteration    3 best reduced chisq so far: 9.662E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7849  0.296538  129.9724  130.0020   10.1985    0.5922    0.1642    1.8599    9.8998
starting iteration    4 best reduced chisq so far: 9.644E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.1262  0.295257  129.9756  130.0021    9.9673    0.5915    0.1769    1.7627    9.2936
starting iteration    5 best reduced chisq so far: 9.640E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0680  0.294488  129.9827  130.0015   10.0086    0.5911    0.1781    1.7385    9.1781
starting iteration    6 best reduced chisq so far: 9.637E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9360  0.293710  129.9916  130.0009   10.0593    0.5910    0.1789    1.6905    8.8266
starting iteration    7 best reduced chisq so far: 9.636E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9697  0.293546  129.9918  130.0008   10.0676    0.5910    0.1778    1.6857    8.7210
starting iteration    8 best reduced chisq so far: 9.636E-01
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9823  0.292840  129.9915  130.0013   10.0497    0.5909    0.1739    1.6705    8.3468

 Done minimization
 Total number of iterations:            8
 Minimum chi^2 found:   0.963588774    

 Wrote:  EXAMPLE/PHOT/OUT/err.mod.fits
 Wrote:  EXAMPLE/PHOT/OUT/err.res.fits
 Wrote:  EXAMPLE/PHOT/OUT/err.dsk.fits
 Wrote:  EXAMPLE/PHOT/OUT/err.bar.fits
 Wrote:  EXAMPLE/PHOT/OUT/err.blg.fits

 Determining uncertainties...
 Beginning bootstrap


 Bootstrap number:            1
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.982E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 1.111E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.8636  0.304491  129.9458  129.9790   12.1363    0.5817    0.1633    1.9046   10.5121
starting iteration    3 best reduced chisq so far: 1.082E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.6659  0.290866  129.9800  129.9794   10.9447    0.5905    0.1881    1.8359   10.1908
starting iteration    4 best reduced chisq so far: 1.080E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9219  0.289887  129.9816  129.9808   10.7078    0.5912    0.1936    1.7968    9.9129
starting iteration    5 best reduced chisq so far: 1.080E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.1020  0.289155  129.9896  129.9865   10.5956    0.5908    0.1982    1.7088    9.1889
starting iteration    6 best reduced chisq so far: 1.080E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0434  0.288464  129.9908  129.9892   10.6374    0.5909    0.1975    1.6955    9.0202
starting iteration    7 best reduced chisq so far: 1.079E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0162  0.287588  129.9872  129.9924   10.6502    0.5909    0.1962    1.6765    8.6583
starting iteration    8 best reduced chisq so far: 1.079E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0140  0.287416  129.9869  129.9915   10.6317    0.5910    0.1950    1.6749    8.5545
starting iteration    9 best reduced chisq so far: 1.079E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0242  0.287381  129.9873  129.9906   10.6191    0.5910    0.1924    1.6708    8.3685

 Done minimization
 Total number of iterations:            9
 Minimum chi^2 found:    1.07940936    



 Bootstrap number:            2
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.974E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 1.103E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.8344  0.306568  129.9463  129.9692   11.9279    0.5840    0.1475    1.9402   10.4606
starting iteration    3 best reduced chisq so far: 1.074E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.6430  0.294451  129.9869  129.9698   10.6744    0.5922    0.1689    1.8867   10.1194
starting iteration    4 best reduced chisq so far: 1.073E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.7664  0.294547  129.9882  129.9697   10.4033    0.5916    0.1762    1.8204    9.6029
starting iteration    5 best reduced chisq so far: 1.072E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.7944  0.294188  129.9924  129.9730   10.4175    0.5914    0.1768    1.8000    9.4745
starting iteration    6 best reduced chisq so far: 1.072E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.7454  0.293466  129.9920  129.9826   10.5039    0.5915    0.1777    1.7389    8.9657
starting iteration    7 best reduced chisq so far: 1.072E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.7684  0.292874  129.9921  129.9823   10.5035    0.5914    0.1767    1.7331    8.8320
starting iteration    8 best reduced chisq so far: 1.072E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.7653  0.291910  129.9930  129.9816   10.4932    0.5906    0.1733    1.7145    8.4065

 Done minimization
 Total number of iterations:            8
 Minimum chi^2 found:    1.07174957    



 Bootstrap number:            3
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.934E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 1.123E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.6763  0.303392  129.9437  130.0018   10.9438    0.5813    0.1443    1.9072   10.3781
starting iteration    3 best reduced chisq so far: 1.105E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.2560  0.295536  129.9725  130.0106   10.0492    0.5877    0.1656    1.8376   10.2663
starting iteration    4 best reduced chisq so far: 1.103E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.5986  0.294723  129.9808  130.0038    9.8029    0.5880    0.1838    1.7487   10.1839
starting iteration    5 best reduced chisq so far: 1.102E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.5615  0.294547  129.9869  130.0030    9.8818    0.5878    0.1870    1.7370   10.3669
starting iteration    6 best reduced chisq so far: 1.102E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.5131  0.294868  129.9908  130.0031    9.9509    0.5877    0.1902    1.7289   10.6011
starting iteration    7 best reduced chisq so far: 1.102E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.5077  0.295046  129.9907  130.0030    9.9509    0.5877    0.1903    1.7321   10.6200

 Done minimization
 Total number of iterations:            7
 Minimum chi^2 found:    1.10209358    



 Bootstrap number:            4
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.901E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 1.130E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.2216  0.310524  129.9469  129.9891   12.1355    0.5850    0.1402    1.9391   10.2957
starting iteration    3 best reduced chisq so far: 1.094E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   38.9263  0.299437  129.9755  129.9929   10.5136    0.5949    0.1608    1.8831    9.8165
starting iteration    4 best reduced chisq so far: 1.092E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.1257  0.299428  129.9733  129.9952   10.3075    0.5948    0.1639    1.8445    9.4260
starting iteration    5 best reduced chisq so far: 1.090E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.3143  0.297446  129.9825  130.0015   10.1565    0.5918    0.1653    1.6627    7.5139
starting iteration    6 best reduced chisq so far: 1.090E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.2381  0.295750  129.9879  129.9999   10.2274    0.5930    0.1604    1.6279    7.1131
starting iteration    7 best reduced chisq so far: 1.090E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.2102  0.295051  129.9904  129.9995   10.2180    0.5934    0.1577    1.6138    6.9476
starting iteration    8 best reduced chisq so far: 1.090E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.1846  0.294811  129.9923  129.9991   10.2144    0.5936    0.1551    1.6006    6.7999

 Done minimization
 Total number of iterations:            8
 Minimum chi^2 found:    1.08974755    



 Bootstrap number:            5
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    1 best reduced chisq so far: 1.866E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   35.0000  0.400000  130.0000  130.0000    5.0000    0.5000    0.2000    2.0000   10.0000
starting iteration    2 best reduced chisq so far: 1.130E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7920  0.316249  129.9420  129.9948   11.9600    0.5807    0.1383    1.9518   10.2778
starting iteration    3 best reduced chisq so far: 1.096E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.7754  0.305510  129.9686  129.9986   10.3375    0.5910    0.1576    1.9032    9.8241
starting iteration    4 best reduced chisq so far: 1.094E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.8999  0.305743  129.9668  129.9998   10.0786    0.5907    0.1599    1.8687    9.4400
starting iteration    5 best reduced chisq so far: 1.093E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0106  0.304906  129.9670  129.9998   10.1086    0.5906    0.1586    1.7266    7.8684
starting iteration    6 best reduced chisq so far: 1.092E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   40.0177  0.302020  129.9773  129.9998   10.0196    0.5903    0.1551    1.6739    7.2702
starting iteration    7 best reduced chisq so far: 1.091E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9920  0.301236  129.9810  130.0003    9.9964    0.5903    0.1519    1.6535    7.0559
starting iteration    8 best reduced chisq so far: 1.091E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9784  0.301019  129.9830  130.0003    9.9919    0.5903    0.1506    1.6431    6.9666
starting iteration    9 best reduced chisq so far: 1.091E+00
   diskpa:   diskel:     xcen:     ycen:   bar-pa:   bar-el:   blg-el:    blg-n:    r-blg:
   39.9724  0.300935  129.9842  130.0001    9.9894    0.5903    0.1503    1.6381    6.9373

 Done minimization
 Total number of iterations:            9
 Minimum chi^2 found:    1.09124732    



 Finished           5  bootstrap iterations
 Wrote out: EXAMPLE/PHOT/OUT/err.out
