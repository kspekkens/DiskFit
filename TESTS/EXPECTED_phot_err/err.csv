disk pa,disk el,xcen,ycen,bar pa,bar el,bulge el,Sersic n,bulge R_e,bulge I_0,f_{disk},f_{bar},f_{bulge}
  4.0022E+01,  2.8718E-01,  1.2999E+02,  1.2999E+02,  1.0617E+01,  5.9101E-01,  1.9227E-01,  1.6631E+00,  8.2762E+00,  3.7520E+02,  7.9964E-01,  9.9893E-02,  1.0047E-01
  4.0759E+01,  2.9206E-01,  1.2999E+02,  1.2998E+02,  1.0487E+01,  5.9105E-01,  1.7228E-01,  1.7143E+00,  8.3884E+00,  3.4904E+02,  7.9530E-01,  1.0498E-01,  9.9727E-02
  3.9505E+01,  2.9504E-01,  1.2999E+02,  1.3000E+02,  9.9535E+00,  5.8769E-01,  1.9042E-01,  1.7343E+00,  1.0650E+01,  3.2977E+02,  7.5552E-01,  9.5669E-02,  1.4881E-01
  3.9185E+01,  2.9481E-01,  1.2999E+02,  1.3000E+02,  1.0217E+01,  5.9361E-01,  1.5515E-01,  1.6006E+00,  6.8092E+00,  3.9565E+02,  8.3229E-01,  9.3750E-02,  7.3956E-02
  3.9974E+01,  3.0095E-01,  1.2998E+02,  1.3000E+02,  9.9877E+00,  5.9030E-01,  1.5038E-01,  1.6363E+00,  6.9408E+00,  3.8470E+02,  8.2525E-01,  9.8875E-02,  7.5870E-02
