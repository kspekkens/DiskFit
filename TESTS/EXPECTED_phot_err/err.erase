Erase this file to exit DiskFit gracefully
