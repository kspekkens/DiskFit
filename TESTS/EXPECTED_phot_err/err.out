Minimization output, phot
-------------------------
Input files:
EXAMPLE/PHOT/phot_err.inp
EXAMPLE/PHOT/phot.fits  pixscale  0.50 arcsec/pix
sky:  800.00  skysig:   14.14    gain:    4.00

FITS region:       x-low:      1   y-low:      1 x-range:    256 y-range:    256
FITS sampling:    regrad:  10.00   regpa:  35.00  regeps:   0.40 istpout:      2

Output model and (data-model) residuals files:
EXAMPLE/PHOT/OUT/err.mod
EXAMPLE/PHOT/OUT/err.res

Disk toggles:         PA: T     eps: T  center: T non-axi: T    phib: T
phot toggles:     lbulge: T    lr_e: T  lsersn: T blg eps: T bar eps: T

Input values
------------
disk PA, phi_d^prime (deg):           35.00
disk eps:                              0.40
x,y center (data units):             130.00  130.00
Bar PA (deg):                          5.00
Bar eps:                               0.50
Bulge r_e (pixels):                   10.00
Sersic n:                              2.00
Bulge eps:                             0.20




Seeing correction applied. FWHM (data units):      2.00
No model component smoothing applied
Uncertainties estimated via bootstrap:  seed:      -50  nunc:        5  junc:   -1.00
Non-axisymmetries fixed to 0 outside range:    5.00   40.00


Best fitting values
-------------------
disk PA, phi_d^prime (deg):           39.98 +/-  0.56
disk eps:                              0.29 +/-  0.00
disk incl (deg):                      44.99 +/-  0.39
x,y center (data units):             129.99 +/-  0.00,  130.00 +/-  0.01
Non-axisymm phib (sky plane, deg):    10.05 +/-  0.28, -129.15
Bar eps:                               0.59 +/-  0.00
Bulge r_e (pixels):                    8.35 +/-  1.42
Bulge I_0 (ADU):                     365.15 +/-    26.25
Sersic n:                              1.67 +/-  0.05
Bulge eps:                             0.17 +/-  0.02

Disk light fraction (%):              79.95 +/-  2.80
Bar light fraction (%):                9.84 +/-  0.40
Bulge light fraction (%):             10.21 +/-  2.75

Minimization Details
---------------------
# points Dn used in fit:               7495
# iterations in minimization:             8
Minimum chi^2 found:               0.963589
Degrees of freedom in fit:             7458

Fitted intensities                                         (radii in pixels, intensities in ADU):
       r        npts       Idisk      eIdisk        Ibar       eIbar     Ibulge      eIbulge
---------------------------------------------------------------------------------------------
      5.00    125.66      648.51      128.99      190.58       20.12      809.28      191.47
     10.00     99.62      634.22       68.95      173.98       12.85      258.82       90.25
     15.00     83.31      528.17       42.88      198.76       10.74      102.93       45.37
     20.00    111.42      442.70       25.35      207.67        6.65       46.06       24.03
     25.00    138.85      365.75       15.24      181.34       10.72       22.27       13.26
     30.00    167.81      297.14       10.25      247.87        0.82       11.40        7.56
     35.00    194.97      251.13        6.01       65.16        7.45        6.09        4.43
     40.00    222.14      200.74        3.23        0.00        1.96        3.37        2.66
     45.00    250.15      166.12        3.37        0.00        0.00        1.92        1.63
     50.00    277.37      134.53        0.65        0.00        0.00        1.12        1.01
     55.00    306.20      111.57        2.09        0.00        0.00        0.67        0.64
     60.00    333.80       89.43        1.61        0.00        0.00        0.41        0.41
     65.00    360.51       74.48        1.93        0.00        0.00        0.25        0.27
     70.00    388.49       60.16        1.99        0.00        0.00        0.16        0.17
     75.00    415.94       49.51        1.92        0.00        0.00        0.10        0.12
     80.00    680.90       38.74        0.55        0.00        0.00        0.06        0.08
     90.00    999.46       29.25        0.19        0.00        0.00        0.03        0.04
    100.00   1103.94        7.84        0.85        0.00        0.00        0.01        0.02
    110.00   1234.45        0.67        0.85        0.00        0.00        0.01        0.01
